### ISSUE TITLE
object used in example but not created there

### ISSUE THREAD

**Issue by RafalSkolasinski** [pydata/pandas#11223]
In [here](http://pandas-docs.github.io/pandas-docs-travis/groupby.html#groupby-with-multiindex) object ``s`` is used in the example but not created on this page (at least in this form).

Above it is created as series
```python

In [8]: s = pd.Series([1, 2, 3, 10, 20, 30], lst)
```

once under  [here](http://pandas-docs.github.io/pandas-docs-travis/groupby.html#groupby-with-multiindex)

```python
In [27]: s
Out[27]: 
first  second
bar    one      -0.575247
       two       0.254161
baz    one      -1.143704
       two       0.215897
foo    one       1.193555
       two      -0.077118
qux    one      -0.408530
       two      -0.862495
dtype: float64
```

is completely different thing.

I find referring to objects created on other manual pages very confusing.

**Comment by TomAugspurger** [comment]
The Series is created on [line 260](https://github.com/pydata/pandas/blame/master/doc/source/groupby.rst#L260) in `doc/source/groupby.rst`, but the creation was hidden. A pull request to make that visible would be great if you could.

**Comment by jreback** [comment]
closed by #11443
